t/apicast-subset-of-services.t

```diff
@@ -1,55 +1,109 @@
-use lib 't';
-use Test::APIcast 'no_plan';
+use Test::APIcast::Blackbox 'no_plan';
 
 run_tests();
 
 __DATA__
 
 === TEST 1: multi service configuration limited to specific service
---- main_config
-env APICAST_SERVICES_LIST=42,21;
---- http_config
-  include $TEST_NGINX_UPSTREAM_CONFIG;
-  lua_package_path "$TEST_NGINX_LUA_PATH";
-  init_by_lua_block {
-    require('apicast.configuration_loader').mock({
-      services = {
-        {
-          id = 42,
-          backend_version = 1,
-          proxy = {
-            api_backend = "http://127.0.0.1:$TEST_NGINX_SERVER_PORT/api-backend/one/",
-            hosts = { 'one' },
-            proxy_rules = {
-              { pattern = '/', http_method = 'GET', metric_system_name = 'one', delta = 1 }
-            }
+--- env eval
+("APICAST_SERVICES_LIST", "42,21")
+--- configuration
+{
+  "services": [
+    {
+      "backend_version": 1,
+      "proxy": {
+        "hosts": [
+          "one"
+        ],
+        "api_backend": "http://test:$TEST_NGINX_SERVER_PORT/",
+        "proxy_rules": [
+          {
+            "http_method": "GET",
+            "delta": 1,
+            "metric_system_name": "one",
+            "pattern": "/"
           }
-        },
-        {
-          id = 11,
-          proxy = {
-            hosts = { 'two' }
-          }
-        }
-      }
-    })
+        ]
+      },
+      "id": 42
+    },
+    {
+      "proxy": {
+        "hosts": [
+          "two"
+        ]
+      },
+      "id": 11
+    }
+  ]
+}
+--- backend
+  location /transactions/authrep.xml {
+    content_by_lua_block { ngx.exit(200) }
+  }
+--- upstream
+  location ~ / {
+     echo 'yay, api backend';
   }
-  lua_shared_dict api_keys 10m;
---- config
-  include $TEST_NGINX_APICAST_CONFIG;
+--- pipelined_requests eval
+["GET /?user_key=1","GET /?user_key=2"]
+--- more_headers eval
+["Host: one", "Host: two"]
+--- response_body eval
+["yay, api backend\n", ""]
+--- error_code eval
+[200, 404]
+
 
+
+=== TEST 2: multi service configuration limited with Regexp Filter
```

Comment:
I think it would be nice to add a third test that combines `APICAST_SERVICES_LIST` and `APICAST_SERVICES_FILTER_BY_URL`. What do you think?